packages/vscode-lit-plugin/package.json

```diff
@@ -25,16 +25,76 @@
 	"main": "./out/extension",
 	"scripts": {
 		"clean": "rimraf out",
-		"vscode:prepublish": "npm run compile",
-		"compile": "npm run clean & tsc -p ./",
+		"vscode:prepublish": "npm run build",
 		"watch": "tsc -watch -p ./",
 		"publish": "vsce publish",
 		"package": "vsce package",
 		"install:safe": "npm install --ignore-scripts --no-package-lock && npm i --no-package-lock",
 		"readme": "readme generate -i readme.blueprint.md -c readme.config.json",
-		"test": "npm run compile && node ./out/test/scripts/test-runner.js",
+		"test": "wireit",
+		"build": "wireit",
+		"test:normal": "wireit",
+		"test:packaged": "wireit",
+		"package-for-test": "wireit",
 		"test-packaged": "vsce package -o ./out/packaged.vsix && rm -rf ../../../packaged-extension/ && mkdir ../../../packaged-extension/ && unzip -qq ./out/packaged.vsix -d ../../../packaged-extension/ && node ./out/test/scripts/test-runner.js ../../../packaged-extension/extension/"
 	},
+	"wireit": {
+		"build": {
+			"dependencies": [
+				"../ts-lit-plugin:build",
+				"../lit-analyzer:build",
+				"../../:copylink"
+			],
+			"command": "tsc --build",
```

Comment:
You'll want `"clean": false` here, otherwise the output will get deleted on every run. If the `.tsbuildinfo` is in `output`, that would work, but there would be no advantage to `--build`. If it's not in `output`, I think it will actually be broken, because `tsc` thinks previous up-to-date output exists that actually doesn't.

So you want `"clean": false` and to include the `.tsbuildinfo` file in `output`. When https://github.com/google/wireit/issues/70 lands, you can then switch to `"clean": "on-delete"`, which will give you the best result because you'll get incremental build on changes + new files, and a clean build on deleted files (which is better than the default behavior of `tsc --build`, which just leaves compiled output for deleted source files lying around).